### PR TITLE
Added 6 functions to the JScripting API:

### PR DESCRIPTION
**Pull request by alynna**
getOnlinePlayers(): string[];
  Returns an array of players currently online.
 getAllPlayers(): string[];
  Returs an array of all players in the servers database.
 tellraw(msg: string): void;
  Tells the string to the trigger player.  Takes JSON input as accepted
  by the tellraw command, complete with most features available to it.
 Server.tellraw(msg: string): void;
  The same as above, except all players on the server are targets.
 serverLog(msg: string): void;
  Sends the string to the server log / console only.
 getAllArgs(): string;
  Returns a string with all arguments given in the command as a single
  string, separated by spaces.  For use when your script wants
  everything that was given as parameters as a single argument.

### CODE REVIEW

File: jscripting/mc.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
```
Comment (olee) [inline jscripting/mc.d.ts:1]: This whole file is autogenerated and should not be modified by hand which it looks like you did.

**Comment by olee** [comment]
Code formatting of Java files is a bit messy

**Comment by yuuka-miya** [comment]
@olee not a major blocker, can be fixed with a pass of the formatter in intellij

File: jscripting/mc.d.ts (hunk 1 above)
Comment (spacebuilder2020) [reply jscripting/mc.d.ts:1]: @olee I was under the impression that the auto generation was never fully implemented?

Comment (olee) [reply jscripting/mc.d.ts:1]: Of course it was!
I mean I took all that time to write that generater which used Javadoc for creating definition files.

Comment (spacebuilder2020) [reply jscripting/mc.d.ts:1]: The comments suggested otherwise.

https://github.com/ForgeEssentials/ForgeEssentials/blob/develop/build.gradle#L228

Comment (spacebuilder2020) [reply jscripting/mc.d.ts:1]: @olee

Comment (olee) [reply jscripting/mc.d.ts:1]: That's just the gradle task - the generation needs to be run from IDE I think - there was some run configuration / Java main file for that.

Comment (olee) [reply jscripting/mc.d.ts:1]: https://github.com/ForgeEssentials/ForgeEssentials/blob/develop/src/main/java/com/forgeessentials/jscripting/TsdGenerator.java

Comment (spacebuilder2020) [reply jscripting/mc.d.ts:1]: ok

**Comment by alynna** [comment]
I will gladly use the tsd generator, I asked about it and was told it 
was never fully implemented.
It will make my life alot easier as I begin to expose much more of the 
FE/Forge API.
All I really need to know is how to trigger the regeneration of the files.

**Comment by spacebuilder2020** [comment]
@olee Can you document the procedure for using the TsdGenerator on the developer documentation
https://github.com/ForgeEssentials/ForgeEssentials/wiki/Developer-documentation

**Comment by AnrDaemon** [comment]
@olee ping

**Comment by spacebuilder2020** [comment]
I am going to go ahead and merge this later today if there are no objections.  Since tsd files have diverged a bit, I may need to regenerate them, but other than that, I think it is fine.

File: src/main/java/com/forgeessentials/jscripting/fewrapper/fe/JsCommandArgs.java [hunk 2]
```diff
@@ -83,12 +84,21 @@ public String peek()
...
     }
-
+    
     public String get(int index)
     {
         return that.get(index);
     }
-
+    
+    public String getAllArgs()
```
Comment (olee) [inline src/main/java/com/forgeessentials/jscripting/fewrapper/fe/JsCommandArgs.java:93]: This function is not really necessary as you can do the same simply in JS.
Also that's kinda nasty code with concatenating and then trimming - there's a join function for that.

I think a better solution would be to add this:
```java
public List<String> asArray()
{
    return (List)that.args;
}
```
Which would then allow in JS `var argString = args.asArray().join(' ')`

**Comment by olee** [comment]
Other than that one thing I mentioned above and regenerating the .d.ts files I think this is fine

**Pull request merged**